addons/sale/models/sale_order_line.py

```diff
@@ -475,6 +476,13 @@ def _get_downpayment_description(self):
 
         return name
 
+    @api.depends('product_id.display_name')
```

Comment:
```suggestion
    @api.depends('product_id')
```
Should only depend on the product, the display_name won't ever be changed in the same request.

Reply:
I just have one question: does it have a performance drawback to depend on the display_name? cause fonctionaly, it does depend on the display_name, and who knows, someone could decide to use that field in anther context 🤷

Reply:
Unless they fixed it in the framework, in the past modifying a dependency of the field would make the orm fetch all the records with that dependency to mark them to recompute, even if they were not recomputed because not stored (or read).